.awslocal.env

```diff
@@ -9,9 +9,17 @@ AWS_REGION=eu-west-2
 AWS_ENDPOINT_OVERRIDE=http://host.docker.internal:4566
 AWS_SIGNATURE_VERSION=v4
 
+AWS_NOTIFICATION_BUCKET_NAME=fsd-notification-bucket
+
 # Import application queue variables
 AWS_SQS_IMPORT_APP_QUEUE_NAME=import-queue.fifo
 AWS_DLQ_IMPORT_APP_QUEUE_NAME=import-dlq.fifo
 AWS_DLQ_MAX_RECIEVE_COUNT=3
 AWS_SQS_IMPORT_APP_PRIMARY_QUEUE_URL=http://host.docker.internal:4566/000000000000/import-queue.fifo
 AWS_SQS_IMPORT_APP_SECONDARY_QUEUE_URL=http://host.docker.internal:4566/000000000000/import-queue-dlq.fifo
+
+AWS_SQS_NOTIFICATION_APP_QUEUE_NAME=notif-queue.fifo
+AWS_DLQ_NOTIFICATION_APP_QUEUE_NAME=notif-dlq.fifo
+AWS_DLQ_NOTIFICATION_MAX_RECIEVE_COUNT=3
+AWS_SQS_NOTIFICATION_APP_PRIMARY_QUEUE_URL=http://host.docker.internal:4566/000000000000/notification-queue.fifo
```

Comment:
@nuwan-samarasinghe  do we add these in secrets too?

Reply:
@NarenderRajuB this is for the local environment & this file will be read by the shell script to create resources in local stack